src/test/scala/mrpowers/jodie/DeltaTestUtils.scala

```diff
@@ -0,0 +1,35 @@
+package mrpowers.jodie
+
+import io.delta.tables.DeltaTable
+
+object DeltaTestUtils extends SparkSessionTestWrapper{
+  def executeMergeFor(tableName: String, deltaTable: DeltaTable, updates: List[(Int, String, Int)]) = {
```

Comment:
For a function in a Utils object, this is too specific maybe we can make it generic by passing a list of DF instead of a list of tuples and an additional parameter with the merge condition? Let me know your thoughts.

Reply:
True. this is very specific. Two reasons :
1. List of tuple helps me control the number of updates I want to make
2. Since these are TestUtils and mostly used in unit tests, wanted to wait and see. I am pretty sure future use cases will help us refactor it in a much better way.

Reply:
Fair enough.